modules/release-notes/pages/index.adoc

```diff
@@ -223,7 +275,10 @@ Previously affected character sets included the Unicode range `0080` to `00A0` a
 * Extended the timeout length for queries in the wait queue so that they are not timed out early.
 
 == Known issues
-
+* An exception error is thrown when "" fail to covert to int64 when using `gcollect` to collect the tsar data.
+* After a global loading job is running for a while a fail can be encountered when getting the loading status due to `KAFKASTRM-LL` not being online, when actually the status is online. Then the global loading process will exit and and fail the local job after timeout while waiting the global loading job to finish.
+* For v3.9.1 and v3.9.2 when inserting a new edge in `GPR` and `INTERPRET` mode, the GPE will print out a warning message, because it calls `graphupdates()` and `getau()` API and the `subtypeid` and discriminator string is not set for new-inserted edges.
```

Comment:
There mentions a lot of things that are not user-facing: GPR, graphupdates(), getau(), subtypeid.
I *think* what this is saying this that a false(?) warning message will be printed when inserting edges if (some condition).  That condition needs to be clarify. Does it correlate to something the user did, or does it depend on internal state which the user has no direct control over?

Reply:
updated this with latest commit.